src/Components/Characterdetails.js

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable */
+import { useSelector } from 'react-redux';
+import { BsBackspace } from 'react-icons/bs';
+import { Link } from 'react-router-dom';
+import Characters from './Characters';
+
+function Characterdetails() {
```

Comment:
- [ ] Let's try to layout better the character's details page. Right now some of the elements are flowing out of the containers and the `back` button is overlaying over others.

Yours:
![image](https://user-images.githubusercontent.com/79114233/147131563-5f7d3f98-2f70-4a2e-8c26-20c74f5486f3.png)

Design reference:
![image](https://user-images.githubusercontent.com/79114233/147130727-5c8ed6ed-ef2e-49cc-ba17-52efa1f9a9d4.png)

Reply:
[OPTIONAL] I think you could also include a different image that adapts for the mobiles views on the homepage, as the current one flows out the page. This will provide a better user experience and will make your project look more polished and professional.

![image](https://user-images.githubusercontent.com/79114233/147131609-363b6658-df0f-43c1-97a1-c07718182883.png)